File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        l = []
        for i, num1 in enumerate(nums):
            num2 = target - num1
            for j in range(i + 1, len(nums)):
                if nums[j] == num2:
                    l.append(i)
                    l.append(j)

                    return l
                    break

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    const result = [];
        for (let i = 0; i < nums.length; i++) {
            const num1 = nums[i];
            const num2 = target - num1;
            for (let j = i + 1; j < nums.length; j++) {
                if (nums[j] === num2) {
                    result.push(i);
                    result.push(j);
                    return result;
                }
            }
        }
        return result;
    }